### DISCUSSION TITLE
Welcome to WalletWasabi Discussions!

### DISCUSSION THREAD

**Discussion by molnard** [molnard/WalletWasabi#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by molnard** [comment]
Hello there.

**Comment by ghost** [comment]
well that's interesting!

**Reply by molnard** [reply]
Yes it is, isn't it?

**Reply by ghost** [reply]
notifications work too.

**Reply by molnard** [reply]
What notification? :)

**Reply by ghost** [reply]
https://github.com/notifications?query=is%3Aunread+
![screenshot_20201210_104407](https://user-images.githubusercontent.com/30683012/101754984-a6274000-3ad4-11eb-80de-cd06b928418c.png)
